plugins/languages/rust/rustaceanvim/default.nix

```diff
@@ -50,13 +50,9 @@ helpers.neovim-plugin.mkNeovimPlugin config {
   };
 
   callSetup = false;
+  configLocation = null;
```

Comment:
Perhaps add a note explaining why we can't just do `configLocation = if config.plugins.lsp.enable then [ "plugins" "lsp" "postConfig" ] else "extraConfigLua"` and that we'll implement it below using `mkIf`?